packages/paste-core/components/table/stories/index.stories.tsx

```diff
@@ -0,0 +1,364 @@
+import {Box} from '@twilio-paste/box';
+import {MediaObject, MediaBody, MediaFigure} from '@twilio-paste/media-object';
+import {Menu, MenuButton, MenuItem, MenuSeparator, useMenuState} from '@twilio-paste/menu';
+import {AttachIcon} from '@twilio-paste/icons/esm/AttachIcon';
+import {MoreIcon} from '@twilio-paste/icons/esm/MoreIcon';
+import {Text} from '@twilio-paste/text';
+import {Truncate} from '@twilio-paste/truncate';
+import {Table, THead, TBody, TFoot, Tr, Td, Th} from '../src';
+
+const MoreMenu: React.FC<{}> = () => {
+  const menu = useMenuState();
+  return (
+    <Box display="flex" justifyContent="flex-end">
+      <MenuButton {...menu} variant="reset" size="reset">
+        <MoreIcon decorative={false} title="show more" />
+      </MenuButton>
+      <Menu {...menu} aria-label="Preferences">
+        <MenuItem {...menu}>
+          <MediaObject verticalAlign="center">
+            <MediaFigure spacing="space20">
+              <AttachIcon decorative={false} title="information" />
+            </MediaFigure>
+            <MediaBody>Settings</MediaBody>
+          </MediaObject>
+        </MenuItem>
+        <MenuItem {...menu} disabled>
+          Extensions
+        </MenuItem>
+        <MenuSeparator {...menu} />
+        <MenuItem {...menu}>Keyboard shortcuts</MenuItem>
+      </Menu>
+    </Box>
+  );
+};
+
+storiesOf('Components|Table', module)
+  .addDecorator(withKnobs)
+  .add('Basic table', () => {
+    return (
+      <Table>
+        <THead>
+          <Tr>
+            <Th>Date</Th>
+            <Th>SID</Th>
+            <Th>Direction</Th>
+            <Th>From</Th>
+          </Tr>
+        </THead>
+        <TBody>
+          <Tr>
+            <Td>
+              <Text as="p">16:24:28 PDT</Text>
+              <Text as="p">2020-09-17</Text>
+            </Td>
+            <Td>
+              <Text as="span" fontFamily="fontFamilyCode">
+                SM0yc4mxi6cn4z13bte7qmflc2drc85mlp
+              </Text>
+            </Td>
+            <Td>Incoming</Td>
+            <Td>(602) 609-6747</Td>
+          </Tr>
+          <Tr>
+            <Td>
+              <Text as="p">16:24:28 PDT</Text>
+              <Text as="p">2020-09-17</Text>
+            </Td>
+            <Td>
+              <Text as="span" fontFamily="fontFamilyCode">
+                SMl29llgoihx286uhxfb0yc5n0sg391x5n
+              </Text>
+            </Td>
+            <Td>Incoming</Td>
+            <Td>(602) 609-6747</Td>
+          </Tr>
+          <Tr>
+            <Td>
+              <Text as="p">16:24:28 PDT</Text>
+              <Text as="p">2020-09-17</Text>
+            </Td>
+            <Td>
+              <Text as="span" fontFamily="fontFamilyCode">
+                SMxarke3v30fv17hauqn86a7nhgm3b5d87
+              </Text>
+            </Td>
+            <Td>Incoming</Td>
+            <Td>(602) 609-6747</Td>
+          </Tr>
+        </TBody>
+      </Table>
+    );
+  })
+  .add('Fixed table', () => {
```

Comment:
The layperson may read this as "not broken". We should be a little more specific that this is a `tableLayout: fixed` Table

Reply:
Changing to `TableLayout: Fixed table`.